po/ja.po

```diff
@@ -106,889 +106,889 @@ msgstr ""
+msgstr "MMIO"
 
 #: src/SUMMARY.md:233
 msgid "Let's Write a UART Driver"
-msgstr ""
+msgstr "UARTドライバ"
 
 #: src/SUMMARY.md:234
 msgid "More Traits"
-msgstr ""
+msgstr "他のトレイト"
 
 #: src/SUMMARY.md:235
 msgid "A Better UART Driver"
-msgstr ""
+msgstr "UARTドライバの改善"
 
 #: src/SUMMARY.md:236
 msgid "Bitflags"
-msgstr ""
+msgstr "ビットフラッグ"
 
 #: src/SUMMARY.md:237
 msgid "Multiple Registers"
-msgstr ""
+msgstr "複数のレジスタ"
 
 #: src/SUMMARY.md:238
 msgid "Driver"
-msgstr ""
+msgstr "ドライバ"
 
 #: src/SUMMARY.md:239 src/SUMMARY.md:241
 msgid "Using It"
-msgstr ""
+msgstr "使用例"
 
 #: src/SUMMARY.md:243
 msgid "Useful Crates"
-msgstr ""
+msgstr "便利クレート"
 
 #: src/SUMMARY.md:244
 msgid "zerocopy"
-msgstr ""
+msgstr "zerocopy"
 
 #: src/SUMMARY.md:245
 msgid "aarch64-paging"
-msgstr ""
+msgstr "aarch64-paging"
 
 #: src/SUMMARY.md:246
 msgid "buddy_system_allocator"
-msgstr ""
+msgstr "buddy_system_allocator"
 
 #: src/SUMMARY.md:247
 msgid "tinyvec"
-msgstr ""
+msgstr "tinyvec"
 
 #: src/SUMMARY.md:248
 msgid "spin"
-msgstr ""
+msgstr "spin"
 
 #: src/SUMMARY.md:250
 msgid "vmbase"
-msgstr ""
+msgstr "vmbase"
 
 #: src/SUMMARY.md:252
 msgid "RTC Driver"
-msgstr ""
+msgstr "RTC（リアルタイムブロック）ドライバ"
 
 #: src/SUMMARY.md:255
 msgid "Concurrency: Morning"
-msgstr ""
+msgstr "並行性：AM"
 
 #: src/SUMMARY.md:260
 msgid "Threads"
-msgstr ""
+msgstr "スレッド"
 
 #: src/SUMMARY.md:261
 msgid "Scoped Threads"
-msgstr ""
+msgstr "スコープ付きスレッド"
 
 #: src/SUMMARY.md:262
 msgid "Channels"
-msgstr ""
+msgstr "チャネル"
 
 #: src/SUMMARY.md:263
 msgid "Unbounded Channels"
-msgstr ""
+msgstr "非同期チャネル"
 
 #: src/SUMMARY.md:264
 msgid "Bounded Channels"
-msgstr ""
+msgstr "同期チャネル"
 
 #: src/SUMMARY.md:265
 msgid "Send and Sync"
-msgstr ""
+msgstr "SendとSync"
 
 #: src/SUMMARY.md:265
 msgid "Send"
-msgstr ""
+msgstr "Send"
 
 #: src/SUMMARY.md:265
 msgid "Sync"
-msgstr ""
+msgstr "Sync"
 
 #: src/SUMMARY.md:268
 msgid "Examples"
-msgstr ""
+msgstr "例"
 
 #: src/SUMMARY.md:269
 msgid "Shared State"
-msgstr ""
+msgstr "状態共有"
 
 #: src/SUMMARY.md:270
 msgid "Arc"
-msgstr ""
+msgstr "Arc"
 
 #: src/SUMMARY.md:271
 msgid "Mutex"
-msgstr ""
+msgstr "Mutex"
 
 #: src/SUMMARY.md:274 src/SUMMARY.md:294
 msgid "Dining Philosophers"
-msgstr ""
+msgstr "食事する哲学者"
 
 #: src/SUMMARY.md:275
 msgid "Multi-threaded Link Checker"
-msgstr ""
+msgstr "マルチスレッド・リンクチェッカー"
 
 #: src/SUMMARY.md:277
 msgid "Concurrency: Afternoon"
-msgstr ""
+msgstr "並行性：PM"
 
 #: src/SUMMARY.md:279
 msgid "Async Basics"
-msgstr ""
+msgstr "Asyncの基礎"
 
 #: src/SUMMARY.md:280
 msgid "async/await"
-msgstr ""
+msgstr "async/await"
 
 #: src/SUMMARY.md:281
 msgid "Futures"
-msgstr ""
+msgstr "Futures"
 
 #: src/SUMMARY.md:282
 msgid "Runtimes"
-msgstr ""
+msgstr "ランタイム"
 
 #: src/SUMMARY.md:283
 msgid "Tokio"
-msgstr ""
+msgstr "Tokio"
 
 #: src/SUMMARY.md:284
 msgid "Tasks"
-msgstr ""
+msgstr "タスク"
 
 #: src/SUMMARY.md:285
 msgid "Async Channels"
-msgstr ""
+msgstr "Asyncチャネル"
 
 #: src/SUMMARY.md:287
 msgid "Join"
-msgstr ""
+msgstr "結合"
```

Comment:
"Join" is a keyword, so should be translated.
cf. https://rust-lang.github.io/async-book/06_multiple_futures/02_join.html